data/lui2020viral.yaml

```diff
@@ -0,0 +1,184 @@
+title: Viral dynamics of SARS-CoV-2 across a spectrum of disease severity in COVID-19
+doi: 10.1016/j.jinf.2020.04.014
+description: >
+  Lui et al. report on a prospective cohort study of patients with variable
+  disease severity. Viral loads for positive samples were extracted from Tbl. 1 in
+  the supplementary material, and negatives were extracted manually from Fig. 1 and
+  a figure in the supplementary material. The level of quantification was extracted
+  from methods in the supplementary material.
+
+  Data for other specimen types, including nasopharyngeal swabs, sputum, plasma, and
+  urine, are also available in the source but have not yet been included here.
+analytes:
+  2019-nCoV_N1:
+    description: Gene copies of the N1 gene quantified using RT-qPCR in stool samples.
+    limit_of_detection: unknown
+    limit_of_quantification: 694.1199999839523
+    specimen: stool
+    gene_target: N1
+    unit: gc/mL
+    reference_event: symptom onset
+    biomarker: SARS-CoV-2
+participants:
+- attributes:
+    severity: critical
+  measurements:
+  - analyte: 2019-nCoV_N1
+    time: 15
+    value: negative
+  - analyte: 2019-nCoV_N1
+    time: 20
+    value: 3861.000000000001
```

Comment:
Why do all the `value` numbers have so many digit points?

Reply:
These are rounding errors because I originally stored the data on the log10 scale in https://github.com/tillahoffmann/shedding. We could round them to fewer digits if desired, but it wouldn't make a difference numerically. What do you think?

Reply:
That's fine. We can keep them. Thank you for explaining.